src/terraform-resource/out/out.go

```diff
@@ -48,6 +48,9 @@ func (r Runner) Run(req models.OutRequest) (models.OutResponse, error) {
 	storageDriver := storage.BuildDriver(storageModel)
 
 	terraformModel := req.Source.Terraform.Merge(req.Params.Terraform)
+	if req.Params.OutputModule != "" {
```

Comment:
Thinking we should remove any reference to `OutputModule` in the `out` steps. As part of `put` the terraform outputs are only used to show metadata in the Concourse build UI. This would also look a bit confusing as you'd have to specify the property on both `get` and `put`:

```yaml
- put: terraform
  params:
    output_module: foo
  get_params:
    output_module: foo
```